tests/src/Common/PerfHarness/PerfHarness.csproj

```diff
@@ -6,9 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.performance.api">
-      <Version>1.0.0-beta-build0006</Version>
-    </PackageReference>
+    <PackageReference Include="xunit.performance.api" Version="1.0.0-beta-build0007" />
```

Comment:
I don't think this works with all versions of msbuild - @weshaggard can you comment?